template/build/utils.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 const path = require('path')
 const config = require('../config')
-const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const packageConfig = require('../package.json')
```

Comment:
Thanks so much for this PR, saved me a lot of pain. Very small thing - `packageConfig` doesn't get used in this file, so this throws an error in eslint 👍